apps/backend/src/api/devices.py

```
@@ -8,41 +8,45 @@
 The core SSH-based monitoring tools work directly with hostnames and don't require device registration.
 """
 
+from datetime import UTC
```

Ruff (0.12.2)

`datetime.UTC` imported but unused

Remove unused import: `datetime.UTC`

(F401)